### ISSUE TITLE
Text isn't "crisp"

### ISSUE THREAD

**Issue by Hayorti** [caiceA/slack-black-theme#36]
I just installed this theme on Slack 3.3.8 on MacOS Mojave (10.14.2) and even though the colours are great, the text doesn't look as crisp as it did without the theme or in your screenshot. It's almost as if Slack or MacOS is having trouble rendering it. I tried installing the font both into User Fonts and System Fonts, but I'm having the same problem.

The difference is subtle, but noticeable when you're looking at a full page of text. I can't provide a full window screenshot, but I've attached a sample that might be clear enough to notice the strangeness. It's especially noticeable when something is in code syntax.

Any ideas?

![Slack](https://user-images.githubusercontent.com/14979514/54937854-a300b080-4efb-11e9-9bae-7f3cd2d8f878.png)

**Comment by caiceA** [comment]
The original mockup has a different font, but the current file on the server and on Github has slack's default font.
I don't use default font TBH I use different font family for my personal use which I feel is better, you can try that if that helps

Download and Install this font family : https://www.fontsquirrel.com/fonts/nunito-sans 
use the attached file and replace it with your current one 
[ssb-interop.js.zip](https://github.com/caiceA/slack-black-theme/files/3005648/ssb-interop.js.zip)

**Comment by Hayorti** [comment]
@caiceA I actually got this resolved. It turns out that there wasn't a problem with the font itself, but the lack of proper anti-aliasing/subpixel rendering on my external, non-Retina monitors.

Screenshots aren't showing the difference, and photos of my monitor aren't picking up the subtle, yet noticeable differences, so I'll just list the changes I made:

1. Add `-webkit-font-smoothing: auto;` to force antialiasing/subpixel rendering
```
body {
  font-family: 'Nunito Sans', sans-serif !important;
  /* src: url('https://fonts.googleapis.com/css?family=Nunito:200,200i,300,300i,400,400i,600,600i,700,700i,800,800i,900,900i'); */
  text-rendering: optimizeLegibility important;
  word-spacing: -1px;
  -webkit-font-smoothing: auto;
}
```

2. Add `font-weight: 200;` and `letter-spacing: 0.5px !important;` to the message bodies. After font smoothing, the message body text was too heavy and the characters were too close, so I needed to thin them out and space them a bit further apart.
```
span.c-message__body,
a.c-message__sender_link,
span.c-message_attachment__media_trigger.c-message_attachment__media_trigger--caption,
div.p-message_pane__foreword__description span {
  color: white;
  font-size: 15px;
  font-weight: 200;
  letter-spacing: 0.5px !important;
}
```

3. Add `font-weight: 200;` and `letter-spacing: 0.5px !important;` for the sidebar for the same reasons as for the message bodies.
```
.p-channel_sidebar {
  font-size: 15px !important;
  font-weight: 200;
}
```

Granted, this doesn't cover everything and there's still text that looks too thick or too crowded, but the message and sidebar text was my priority, so those are the only two I worked on.

It definitely looks much more usable now though.
